streams/src/main/java/org/apache/kafka/streams/processor/internals/TaskManager.java

```diff
@@ -223,10 +223,7 @@ boolean handleCorruption(final Set<TaskId> corruptedTasks) {
             final Collection<Task> tasksToCommit = allTasks()
```

Comment:
We could still use `tasks.allTasks()` here, since we certainly do not want to process tasks owned by the state updater right? Would seem cleaner to me.

Reply:
Actually, we want to commit (actually checkpoint) non-corrupted standby tasks which are owned by the state updater.

Reply:
Oh, right. Forgot the checkpointing is piggy-backed here.

Reply:
Actually, not so sure I understand this. Why do we want to checkpoint non-corrupted standby tasks here? The comment says `since this will force the ongoing txn to abort`, but what do standby tasks have to do with the ongoing txn?

Reply:
I am not sure if we strictly need to do it, because as you say standby tasks have nothing to do with the ongoing transaction. I was merely referring to what the old code path does.